microservices/rabbitmq/rabbitmq.conf

```diff
@@ -16,3 +16,5 @@ management.listener.port = 15672
 
 # Enable the management plugin
 management.load_definitions = /etc/rabbitmq/definitions.json
+
+heartbeat = 60
```

Comment:
Is this heartbeat var used somewhere?
Below in DialConfig heartbeat is set to 10 secs but this var says 60secs.

Reply:
@irakr changed to 10 secs